.github/workflows/ci.yml

```diff
@@ -33,6 +34,7 @@ jobs:
       - uses: actions-rs/cargo@v1
         with:
           command: test
+          args: --all-features
```

Comment:
Could you run tests with both `--all-features` and no feature (or `--no-default-features`)?
Often a test succeeds when all features are enabled, but fails when features are restricted.

Reply:
Added some CIs 354d3ca